components/Form/Select/Nationality.jsx

```diff
@@ -0,0 +1,58 @@
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import formData from '../../../data/formData';
```

Comment:
This is not pointing to any file 😅 
Also, we have `baseUrl` in place, so if a file is in the root folder you can just use `data/`, I think it's more clear :)

ps. sorry for being pedantic 😅